FindOnPage.md

```diff
@@ -0,0 +1,478 @@
+ 
+    // Checks if a custom user interface is desired by end developer.
+    // If TRUE, the default Find bar is not displayed.
+    // Returns TRUE if using a custom UI, FALSE if using the default.
+    [propget] HRESULT SuppressDefaultDialog([out, retval] BOOL* value); 
+    // Sets whether to use a custom UI for the Find operation.
+    [propput] HRESULT SuppressDefaultDialog([in] BOOL value);
+
+    // Gets the index of the currently active match.
+    // If there's no active find session but an attempt is made to get the active match index:
+    // The function will return -1.
+    // Returns The active match index.
+    [propget] HRESULT ActiveMatchIndex([out, retval] LONG* value); 
+
+    // Gets the total count of matches found in the current document based on the last find criteria.
+    // Returns the total count of matches.
+    [propget] HRESULT MatchesCount([out, retval] LONG* value);
+
+}
+
+/// Handles the event that's fired when the match count changes.
+[uuid(623EFBFB-A19E-43C4-B309-D578511D24AB), object, pointer_default(unique)]
+interface ICoreWebView2FindMatchCountChangedEventHandler : IUnknown {
+    /// Parameter is the match count.
+    HRESULT Invoke(LONG matchesCount);
+}
+
+/// Handles the event that's fired when the active match index changes.
+[uuid(623EFBF9-A19E-43C4-B309-D578511D24A9), object, pointer_default(unique)]
+interface ICoreWebView2FindActiveMatchIndexChangedEventHandler : IUnknown {
+    /// Parameter is the active match index.
+    HRESULT Invoke(LONG activeMatchIndex);
+}
+/// Handles the event that's fired when the find operation completes.
+[uuid(2604789D-9553-4246-8E21-B9C74EFAD04F), object, pointer_default(unique)]
+interface ICoreWebView2FindOperationCompletedHandler : IUnknown {
+    /// Provides the event args when the find operation completes.
+    HRESULT Invoke(HRESULT errorCode, BOOL status);
+}
+
+// Interface providing methods and properties for finding and navigating through text in the web view.
+[uuid(7C49A8AA-2A17-4846-8207-21D1520AABC0), object, pointer_default(unique)] 
+interface ICoreWebView2Find : IUnknown {
+ 
+    // Initiates a find using the specified configuration.
+    HRESULT StartFind([in] ICoreWebView2FindConfiguration* configuration, 
+                      ICoreWebView2FindOperationCompletedHandler* handler);
+
+    // Navigates to the next match in the document.
+    HRESULT FindNext();
+
+    // Navigates to the previous match in the document.
+    HRESULT FindPrevious();
+
+    // Stops the current 'Find' operation and hides the Find bar.
+    HRESULT StopFind();
+
+    // Registers an event handler for the FindCompleted event.
+    // This event is raised when a find operation completes, either by finding all matches, navigating to a match, or by being stopped.
+    // Parameter is the event handler to be added.
+    // Returns a token representing the added event handler. This token can be used to unregister the event handler.
+    HRESULT add_MatchCountChanged(
```

Comment:
more inconsistency between MatchesCount and MatchCount. MatchCount maybe sounds better?